tests/test_lhotse.py

```diff
@@ -0,0 +1,62 @@
+import json
+
+import pytest
+
+import torchaudio
```

Comment:
@pzelasko Currently, the CI test fails because `torchaudio` is not a dependency of NeMo and does not get installed. Could you please use some other library, maybe `soundfile`?

Reply:
Should be fixed now

Reply:
Ah, now the CI test fails because Lhotse requires torchaudio. I guess lets add torchaudio to the test requirements, with a comment that it is needed for Lhotse?
@pzelasko

Reply:
Done, let's see